app/controllers/users_controller.rb

```diff
@@ -0,0 +1,41 @@
+class UsersController < ApplicationController
+    skip_before_action :authorized, only: [:create, :index]
+    rescue_from ActiveRecord::RecordInvalid, with: :handle_invalid_record
+   
+    def index
+        @users = User.all
+        render json: @users
+    end
+
+    def create 
+        user = User.create!(user_params)
```

Comment:
What will happen if any validation gets failed here ?  Please complete error handling for user creation